### PR TITLE
267988: Database Permission Management and Migration for PostgresDB Flux Manifests

### PR DESCRIPTION
**Pull request by aadev1**
This PR contains auto generated files for the required configuration changes for Database Permission Management and Migration for PostgresDB in all environments for ffc-demo

[AB#267988](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/267988)
[AB#287824](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/287824)

# Testing
All Flux manifests have successfully reconciled.
image

# Checklist (please delete before completing or setting auto-complete)
- [x] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [x] Title pattern should be `{work item number}: {title}`
- [x] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests

# **How does this PR make you feel**:
gif

### CODE REVIEW

**Review by jeevankuduvaravindran — DISMISSED** [review]
ClappingLeonardoDicaprioGIF

File: services/environments/snd/base/image-update.yaml [hunk 1]
```diff
@@ -12,14 +12,14 @@ spec:
   git:
     checkout:
       ref:
-        branch: main
+        branch: aa/testDbMigration
```
Comment (ganeshg2491) [inline services/environments/snd/base/image-update.yaml:15]: branch =main?

File: services/environments/snd/base/image-update.yaml [hunk 2]
```diff
@@ -12,14 +12,14 @@ spec:
...
     commit:
       author:
         email: fluxcdbot@users.noreply.github.com
         name: fluxcdbot
       messageTemplate: '{{range .Updated.Images}}{{println .}}{{end}}'
     push:
-      branch: main
+      branch: aa/testDbMigration
```
Comment (ganeshg2491) [inline services/environments/snd/base/image-update.yaml:22]: branch =main?

File: services/environments/snd/base/image-update.yaml (hunk 1 above)
Comment (aadev1) [reply services/environments/snd/base/image-update.yaml]: Good spot again.  Updated to main

**Review by jeevankuduvaravindran — APPROVED** [review]

**Review by jeevankuduvaravindran — APPROVED** [review]

**Review by kbabigumira — APPROVED** [review]

**Pull request merged**